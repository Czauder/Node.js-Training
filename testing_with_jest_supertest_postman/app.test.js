const request = require('supertest')

const {
  app
} = require('./app');


it('works', async () => {
  const response = await request(app).get('/');
  expect(response.status).toEqual(200);
})

it('hanldes pages are no found', async () => {
  const response = await request(app).get('/whatever');

  expect(response.status).toEqual(404);
  expect(response.text).toEqual('Not found')
})


