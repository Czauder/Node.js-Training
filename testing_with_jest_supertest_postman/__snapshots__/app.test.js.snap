// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hanldes pages are no found 1`] = `
"<!DOCTYPE html>
<html lang=\\"en\\">
<head>
<meta charset=\\"utf-8\\">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /whatever</pre>
</body>
</html>
"
`;
